src/internal/testpachd/real_env.go

```diff
@@ -127,10 +128,10 @@ func NewRealEnv(t testing.TB, customOpts ...serviceenv.ConfigOption) *RealEnv {
 	realEnv.MockPPSTransactionServer = NewMockPPSTransactionServer()
 	realEnv.ServiceEnv.SetPpsServer(&realEnv.MockPPSTransactionServer.api)
 	realEnv.MockPPSTransactionServer.InspectPipelineInTransaction.
-		Use(func(txnctx *txncontext.TransactionContext, name string) (*pps.PipelineInfo, error) {
+		Use(func(txnctx *txncontext.TransactionContext, pipeline *pps.Pipeline) (*pps.PipelineInfo, error) {
 			return nil, col.ErrNotFound{
 				Type: "pipelines",
-				Key:  name,
+				Key:  fmt.Sprintf("%s/%s", pipeline.Project.GetName(), pipeline.Name),
```

Comment:
This seems like it should be a utility function.

Reply:
How do you feel about doing that in the other half of CORE-1039, in PR #8156?